TransferMethod/Sources/UpdateTransferMethodController.swift

```diff
@@ -0,0 +1,473 @@
+        let label = UILabel()
+        label.numberOfLines = 0
+        label.adjustsFontForContentSizeCategory = true
+        let stackView = UIStackView(arrangedSubviews: [label])
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.isLayoutMarginsRelativeArrangement = true
+        stackView.layoutMargins = UIEdgeInsets(
+            top: CGFloat(8.0),
+            left: CGFloat(0.0),
+            bottom: CGFloat(8.0),
+            right: CGFloat(16.0)
+        )
+        return stackView
+    }()
+
+    override public func viewDidLoad() {
+        super.viewDidLoad()
+        initializePresenter()
+        setupTableView()
+        presenter.loadTransferMethodUpdateConfigurationFields()
+        hideKeyboardWhenTappedAround()
+        self.navigationController?.presentationController?.delegate = self
+    }
+
+    override public func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        tableView.reloadData()
+        let currentNavigationItem: UINavigationItem = tabBarController?.navigationItem ?? navigationItem
+        currentNavigationItem.backBarButtonItem = UIBarButtonItem.back
+    }
+
+    override func willMove(toParent parent: UIViewController?) {
+        super.willMove(toParent: parent)
+        if parent == nil {
+            removeCoordinator()
+        }
+    }
+
+    private func setupTableView() {
+        tableView = UITableView(frame: view.frame, style: .grouped)
+        tableView.allowsSelection = false
+        tableView.rowHeight = UITableView.automaticDimension
+        tableView.estimatedRowHeight = Theme.Cell.smallHeight
+        tableView.backgroundColor = Theme.UITableViewController.backgroundColor
+        tableView.accessibilityIdentifier = "updateTransferMethodTable"
+        registeredCells.forEach {
+            tableView.register($0.type, forCellReuseIdentifier: $0.id)
+        }
+    }
+
+    private func initializePresenter() {
+        if let transferMethod = initializationData?[InitializationDataField.transferMethodToken] as?
+            String {
+            presenter = UpdateTransferMethodPresenter(self, transferMethod)
+        } else {
+            fatalError("Required data not provided in initializePresenter")
+        }
+    }
+}
+
+// MARK: - TableViewController Data source and delegate
+extension UpdateTransferMethodController {
+    /// Returns the title for header
+    override public func tableView(_ tableView: UITableView, titleForHeaderInSection section: Int) -> String? {
+        return presenter.sectionData[section].header
+    }
+    /// Returns the header view
+    override public func tableView(_ tableView: UITableView,
+                                   willDisplayHeaderView view: UIView,
+                                   forSection section: Int) {
+        guard let headerView = view as?  UITableViewHeaderFooterView
+            else {
+                return
+        }
```

Comment:
is this required? why not just use `view?.textLabel?.textColor = Theme.Label.subtitleColor`

Reply:
We are casting UIView to UITableViewHeaderFooterView to set the textLabel text color.